### DISCUSSION TITLE
Could I know how to bind other mouse keys but not only 2 in Logitech G602?

### DISCUSSION THREAD

**Discussion by Timzhouyes** [noah-nuebling/mac-mouse-fix#626] [Questions]
Hi!
I have one G602 mouse, and this is an awesome app that can make my mouse alive on Mac. But I wanna know that why only button 4 and button 5 can be recognized but for other keys, from button 6 to button 9 the app cannot trigger. 
Thanks so much for helping!

**Comment by AndyBoWu** [comment]
Having the same issue!! @Timzhouyes 

Bump this up.

**Comment by NiftyMaker** [comment]
Having the same issue too... somehow, macOS detects when I click button 6 to 9 because a sound is triggered, but the app doesn't recognize any of these buttons, and Logitech app doesn't support this mouse on macOS. Also button 4 is recognized as button 5, and vice versa with 5 as 4
